services/docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
       - DIRECTOR_V2_HOST=${DIRECTOR_V2_HOST}
       - DIRECTOR_V2_PORT=${DIRECTOR_V2_PORT}
 
-      - API_SERVER_LOGLEVEL=${API_SERVER_LOGLEVEL:-${LOG_LEVEL:-WARNING}}
+      - API_SERVER_LOGLEVEL=${API_SERVER_LOGLEVEL:-WARNING}
```

Comment:
this is invalid? I think we always had it this way

Reply:
Check for that string in the repo .. you will see it is also used in autoscaling/test/manua/docker-compose ....

Reply:
- We used `docker compose config` in the past to concatenate and subsitute our compose files
- This never supported docker swarm, but only docker without swarm mode
- @sanderegg mainly and in parts me built boilerplate scripts with `sed` to mitigate the shortcommings, probably you can still find this script in `osparc-simcore`
- With the docker engine version upgrade our makeshift tooling broke again, and it showed that now docker has a `config` CLI that supports swarm-mode, which is `docker stack config`
- `docker stack config`, the new CLI command, does not support nested defaults (anymore?)

Reply:
check for which string specifically please?

Reply:
`:-${LOG_LEVEL:-WARNING}}`

Reply:
The script you used in osparc-config-blahblah comes form this repository. So I would suggest to change the script here and make it work to have this whole stuff running again correctly everywhere.
Otherwise we have half-backed solutions and we will continue creating wrong stuff. Let's check tomorrow

Reply:
as discussed in person, with our setup we are not in the position to use `--skip-interpolation`. The `sed` command mitigates the issues that occur in the double interpolation occuring with subsequent uses of `docker stack compose` and `docker stack deploy`

Reply:
Could it be that this difference in how the substitution of strings is done is related to the issue several of us have encountered with the VS code terminal? What I am wondering is if there if the docker plugin in vs code is doing something funny to the behavior of the terminal there? I am just wondering because the issues I encountered there were exactly of this type: the strings were not substituted correctly 🤔

Reply:
Could it be that this difference in how the substitution of strings is done is related to the issue several of us have encountered with the VS code terminal? What I am wondering is if there if the docker plugin in vs code is doing something funny to the behavior of the terminal there? I am just wondering because the issues I encountered there were exactly of this type: the strings were not substituted correctly 🤔

Reply:
I think we agreed to avoid defaults here